section_to_course/models.py

```diff
@@ -0,0 +1,33 @@
+"""
+Database models for section_to_course.
+"""
+# from django.db import models
+from django.db import models
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import CourseKeyField, UsageKeyField
+
+
+class SectionToCourseLink(TimeStampedModel):
+    """
+    A model that links a section to a course.
+
+    .. no_pii:
+    """
+
+    source_course_id = CourseKeyField(max_length=255, db_index=True, null=False, blank=False)
+    destination_course_id = CourseKeyField(max_length=255, db_index=True, null=False, blank=False)
+    source_section_id = UsageKeyField(max_length=255, db_index=True, null=False, blank=False)
+    destination_section_id = UsageKeyField(max_length=255, db_index=True, null=False, blank=False)
+    last_refresh = models.DateTimeField(null=True, blank=True)
```

Comment:
This is a `TimeStampedModel` with the `modified` date. This is redundant.

Reply:
@Agrendalath I'm not sure it will always be. There is a strong potential later on in the project to annotate this model with more metadata, in which case the rest of the fields will be covered by 'modified' rather than 'last_refresh'.